tasks/skylab/Picard.wdl

```diff
@@ -287,3 +287,69 @@ task CollectDuplicationMetrics {
   }
 }
 
+# Here we use "-XX:ParallelGCThreads=2" to run MarkDuplication on multiple threads 
+task RemoveDuplicatesFromBam {
+  input {
+    File aligned_bam
+    String input_id
+    String output_basename
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-picard:v0.2.2-2.10.10"
```

Comment:
Isn't the picard docker published automatically by the picard repository? Maybe we can use the official public docker here instead of our own?

Reply:
I think she means the officially built and released picard docker:
    docker: "us.gcr.io/broad-gotc-prod/picard-cloud:2.25.5"

Reply:
Done. Thank you!